### ISSUE TITLE
Show information on incoming links for a GND entity

### ISSUE THREAD

**Issue by acka47** [hbz/lobid-gnd#142]
Currently we only give information about outgoing links because that is what we have in the data. We should somehow also give an impression about the links that point to the entity, e.g. for a place how many and which people were born or died there, which conferences where held there and which organsations have their place of business in the place.

This can be done in different ways, as lists (that are collapsed by default) or probably also as a visualization using Elasticsearch graph functionality plus Kibana (see #127 ).

**Comment by larsgsvensson** [comment]
Yes, that would be great! Particularly a visualisation of things like narrower terms for topical headings, related terms (and cases with no back link) etc., would be an excellent feature.

**Comment by acka47** [comment]
@Phu2 asked on Mastodon to add a link to administrative inferior organizations to an organization, e.g. add http://lobid.org/gnd/search?q=hierarchicalSuperiorOfTheCorporateBody.id%3A%22http%3A%2F%2Fd-nb.info%2Fgnd%2F10029922-2%22 to http://lobid.org/gnd/10029922-2.
